src/components/PaymentLog.vue

```diff
@@ -1240,7 +1240,8 @@ export default {
             })
             .catch(err => {
               this.$LIPS(false)
-              Flash.setError("Error: " + err.message)
+              Flash.setError("Error: " + err.response?.data?.data?.errors?.serial_number);
```

Comment:
what about other errors? looks like this is only taking care of serial number

Reply:
fixed